src/config/dynamic-theme-fixes.config

```diff
@@ -151,6 +151,18 @@ span[class^="skycon swip"]
 
 ================================
 
+app.datadoghq.com
+
+INVERT
+.HostMap-canvas
+
+CSS
+.log-dt-event.active, .log-dt-event.active:hover, .log-dt-event:hover {
+    background-color: rgb(37, 45, 58);
```

Comment:
Please, check, how it looks in light mode. Most likely the fix should look like `background-color: ${white} !important;` where `${white}` is a pattern containing source color value, and `!important` prevents overrides by other rules.

Reply:
Unfortunately the proposed {white} made it unreadable in light mode, and did not change the original behavior in dark.. Added "!important"